save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt

```diff
@@ -16,12 +18,16 @@ class TestController {
     @Autowired
     private lateinit var testService: TestService
 
+    @Autowired
+    private lateinit var testExecutionService: TestExecutionService
+
     /**
      * @param tests
      */
     @PostMapping("/initializeTests")
-    fun initializeTests(@RequestBody tests: List<Test>) {
-        testService.saveTests(tests)
+    fun initializeTests(@RequestBody testDtos: List<TestDto>) {
+        testService.saveTests(testDtos)
+        testExecutionService.saveTestExecution(testDtos)
```

Comment:
You can pass ids of saved tests from the first method into the second to avoid additional select